src/link/link.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { TestBed, ComponentFixture } from "@angular/core/testing";
+import { Component, Input, DebugElement } from "@angular/core";
+import { By } from "@angular/platform-browser";
+
+import { Link } from "./link.directive";
+
+describe("Link", () => {
+	it("should create a Link", () => {
+		TestBed.configureTestingModule({
```

Comment:
Use `beforeEach` for setting up testing environment by putting common/repeating things into them. See other components' tests for reference.

Reply:
The testbed configuration for the two tests are different. One is using `TestLinkComponent`, the other is using `TestDisabledLinkComponent`.